src/app/shared/components/join-us/join-us.component.scss

```diff
@@ -104,6 +104,10 @@ form {
   text-transform: uppercase;
 }
 
+u {
+  cursor: pointer;
+}
+
 @include laptop {
   .main-container {
     padding: 0;
```

Comment:
Empty lines before each statement.
Tags should be in the beginning of the file (form, input...)
Duplication of class '.info'